### PR TITLE
#163497322 Feature Create Delete Party

### PR DESCRIPTION
**Pull request by akinyeleolat**
### What does this PR do?
delete political party

### Description of Task to be completed?
DELETE /api/v1/parties/partyId is working now

### How should this be manually tested?
- clone repo
- npm install
- npm start and test on postman
### Any background context you want to provide?
NA

### What are the relevant pivotal tracker stories?
163497322

Screenshots (if appropriate)
Questions:

### CODE REVIEW

File: server/controllers/party.js [hunk 1]
```diff
@@ -103,7 +111,38 @@ static updateParty(req, res) {
...
+    const partyDetails = party.find(c => c.id === partyId);
+    if (!partyDetails) {
+      res.status(404).send({
+        status: 404,
+        message: 'The party with given id was not found',
+      });
+      return;
+    }
+    const filteredParty = party.filter(parties => parties !== partyDetails);
+    // const filteredParty = party.slice(0, partyId-1).concat(party.slice(partyId, party.length))
```
Comment (Abudu-Samuel) [inline server/controllers/party.js]: Hi @akinyeleolat try and do away with commented out code when pushing

Comment (Abudu-Samuel) [inline server/controllers/party.js]: Hi @akinyeleolat try and do away with commented out code when pushing

**Review by Abudu-Samuel — CHANGES_REQUESTED** [review]

File: server/controllers/party.js (hunk 1 above)
Comment (akinyeleolat) [reply server/controllers/party.js]: Noted

Comment (Abudu-Samuel) [reply server/controllers/party.js]: LGTM

**Pull request merged**